python/ray/air/checkpoint.py

```diff
@@ -524,6 +524,25 @@ def _save_checkpoint_metadata_in_directory(self, path: str) -> None:
         with open(checkpoint_metadata_path, "wb") as file:
             pickle.dump(self._metadata, file)
 
+    def _copy_dir_ignore_conflict(self, src_dir: Path, dst_dir: Path):
+        """This is a workaround for python < 3.8 where shutil.copytree does not
+        support dirs_exist_ok=True.
```

Comment:
Add a TODO to replace this thing once py37 support is removed.

Reply:
done